lib/action_client/submission_job.rb

```diff
@@ -0,0 +1,33 @@
+module ActionClient
+  class SubmissionJob < ActiveJob::Base
+    def self.after_perform(with_status: nil, **options, &block)
+      if with_status.present?
+        super(
+          if: ->() {
+            status, * = @response
+
+            http_statuses = Array(with_status).map do |status_code|
+              Rack::Utils.status_code(status_code)
+            end
+
+            http_statuses.any? { |code| Array(code).include?(status) }
```

Comment:
Could this be `http_statuses.include?(status)` instead? Is there a situation where `http_statuses` would be an array of arrays?

From a quick look at `Rack::Utils.status_code`, it seems to always return a single value.

Reply:
Done and re-used in https://github.com/thoughtbot/action_client/pull/5/commits/09d3479e3732c69bbdb31225fa70f6f9bc548ddf